### PR TITLE
Tronbox migration & addition of mintable/burnable token contracts

### PR DESCRIPTION
**Pull request by wevans-tron**
Configured migration to tronbox and added mintable/burnable token contracts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,30 @@ Solidity smart contracts for the TRON blockchain https://tron.network
...
+Configured with tronbox https://github.com/tronprotocol/tron-box
+
 Contracts include but not limited to implementations of standard-compliant tokens, library kits, and common utilities.
 
 ## Usage
 
-Please refer to our [Quick Start Guide](https://developers.tron.network/docs/getting-started-1) for testing on a privatenet, or use this [Smart Contract Guide](https://developers.tron.network/docs/deploying) for testing on the Shasta testnet.
+### Install tronbox:
+
+npm install -g tronbox
```
Comment (taiyangc) [inline README.md]: add some code formatting here and the commands below

```
npm install -g tronbox
```

File: README.md [hunk 2]
```diff
@@ -3,11 +3,30 @@ Solidity smart contracts for the TRON blockchain https://tron.network
 
 This repository contains a library of sample smart contracts for the TRON network.
 
+Configured with tronbox https://github.com/tronprotocol/tron-box
```
Comment (taiyangc) [inline README.md]: use [TronBox](https://github.com/tronprotocol/tron-box)

File: contracts/Migrations.sol [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  function setCompleted(uint completed) public restricted {
+    last_completed_migration = completed;
+  }
+
+  function upgrade(address new_address) public restricted {
+    Migrations upgraded = Migrations(new_address);
+    upgraded.setCompleted(last_completed_migration);
+  }
+}
```
Comment (taiyangc) [inline contracts/Migrations.sol]: did this come from tronbox or openzeppelin? is this needed?

File: README.md [hunk 4]
```diff
@@ -3,11 +3,30 @@ Solidity smart contracts for the TRON blockchain https://tron.network
...
+### Compile:
+
+tronbox compile 
+
+### Migrate:
+
+tronbox migrate
+
+
+
```
Comment (taiyangc) [inline README.md]: extra spaces

File: tronbox.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+      from: 'TPL66VK2gCXNCD7EJg9pgJRfqcRazjhUZY',
+      privateKey: 'da146374a75310b9666e834ee4ad0866d6f4035967bfc76217c5a495fff9f0d0',
+      consume_user_resource_percent: 30,
+      fee_limit: 100000000,
+      host: "https://api.trongrid.io",
+      port: 8090,
+      fullNode: "https://api.shasta.trongrid.io",
+      solidityNode: "https://api.shasta.trongrid.io",
+      eventServer: "https://api.shasta.trongrid.io",
+      network_id: "*" // Match any network id
```
Comment (taiyangc) [inline tronbox.js]: this config is a bit out of date, see https://github.com/tronprotocol/tron-box/tree/master/tron-box%20DAPP%20Development#3-configure-network-information

File: tronbox.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  networks: {
+    development: {
+      from: 'TPL66VK2gCXNCD7EJg9pgJRfqcRazjhUZY',
+      privateKey: 'da146374a75310b9666e834ee4ad0866d6f4035967bfc76217c5a495fff9f0d0',
```
Comment (taiyangc) [inline tronbox.js]: this is already well known.. but good to leave it out

File: contracts/Migrations.sol (hunk 3 above)
Comment (wevans-tron) [reply contracts/Migrations.sol]: This comes with Tronbox and is needed for configuration.

**Pull request merged**